### ISSUE TITLE
Duplicated database field?

### ISSUE THREAD

**Issue by kbwestfall** [sdss/marvin#540]
**Describe the bug**
marvin.core.exceptions.MarvinError: API Query call failed: Something went wrong on the server side: Query failed with ValueError: Encountered duplicate field name: 'spaxelprop_specindex_d4000'.

**To Reproduce**
Here's the code:
```
    from marvin.tools.query import Query
    filt = 'spx_snr > 10 and spx_snr < 11 and stellar_sigma > 95 and stellar_sigma < 110 ' \
           'and bintype.name == HYB10 and d4000 > 1.2 and d4000 < 2.8'
    returnparams = ['cube.plateifu', 'spaxelprop.spx_snr', 'spaxelprop.emline_gflux_ha_6564',
                    'spaxelprop.specindex_d4000']
    q = Query(searchfilter=filt, returnparams=returnparams, limit=15000)
    r = q.run()
```

**Expected behavior**
Here's the traceback
```
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/marvin/tools/query.py", line 836, in run
    ii = Interaction(route=url, params=params, stream=True)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/marvin/extern/marvin_brain/python/brain/api/api.py", line 51, in __init__
    self._sendRequest(request_type)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/marvin/extern/marvin_brain/python/brain/api/api.py", line 280, in _sendRequest
    self._checkResponse(self._response)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/marvin/extern/marvin_brain/python/brain/api/api.py", line 245, in _checkResponse
    raise BrainError('Something went wrong on the server side: {0}'.format(errorMsg))
brain.core.exceptions.BrainError: Something went wrong on the server side: Query failed with ValueError: Encountered duplicate field name: 'spaxelprop_specindex_d4000'.
You can submit this error to Marvin GitHub Issues (https://github.com/sdss/marvin/issues/new).
Fill out a subject and some text describing the error that just occurred.
If able, copy and paste the full traceback information into the issue as well.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "./marvin_sel.py", line 22, in <module>
    r = q.run()
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/marvin/tools/query.py", line 99, in wrapper
    return f(self, *args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/marvin/tools/query.py", line 110, in wrapper
    return f(self, *args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/marvin/tools/query.py", line 88, in wrapper
    return f(self, *args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/marvin/tools/query.py", line 840, in run
    raise MarvinError('API Query call failed: {0}'.format(e))
marvin.core.exceptions.MarvinError: API Query call failed: Something went wrong on the server side: Query failed with ValueError: Encountered duplicate field name: 'spaxelprop_specindex_d4000'.
You can submit this error to Marvin GitHub Issues (https://github.com/sdss/marvin/issues/new).
Fill out a subject and some text describing the error that just occurred.
If able, copy and paste the full traceback information into the issue as well..
You can submit this error to Marvin GitHub Issues (https://github.com/sdss/marvin/issues/new).
Fill out a subject and some text describing the error that just occurred.
If able, copy and paste the full traceback information into the issue as well.
Sentry is attempting to send 1 pending error messages
Waiting up to 10 seconds
Press Ctrl-C to quit
```

**Desktop (please complete the following information):**
 - OS: OSX 10.10
 - Version:
```
Python 3.6.2 (v3.6.2:5fd33b5926, Jul 16 2017, 20:11:06)
marvin.__version__
'2.2.6'
```

**Comment by havok2063** [comment]
@kbwestfall It's because you included the `d4000` parameter in the list of return parameters.  Parameters that you specify in the filter automatically get included as a return parameter so you don't need to specify them.  By the way, this duplicate issue is a known bug that has been fixed in the 2.3.0dev version.   If you would you like to be an unofficial vetter for the dev version that would be great.  I'll go ahead and close this.

**Comment by havok2063** [comment]
But I like how you put the traceback in the section of expected behavior.
